EFCore.Seeder.Console/Program.cs

```diff
@@ -9,7 +9,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            var csvConfiguration = new CsvHelper.Configuration.Configuration
+            var csvConfiguration = new CsvHelper.Configuration.CsvConfiguration(new System.Globalization.CultureInfo("en-US")) // .Configuration
```

Comment:
Noticed the new change in the API that requires cultureInfo to be passed in the CsvConfiguration